pkgs/tools/graphics/graphviz/base.nix

```diff
@@ -41,11 +41,16 @@ stdenv.mkDerivation {
     inherit sha256 rev;
   };
 
-  nativeBuildInputs = [ autoreconfHook pkg-config python3 ];
+  nativeBuildInputs = [
+    autoreconfHook
+    pkg-config
+    python3
+    bison
+    flex
+  ];
 
   buildInputs = [
-    libpng libjpeg expat bison libtool fontconfig gd gts libdevil flex pango
-    gettext
+    libpng libjpeg expat fontconfig gd gts libdevil pango bash
   ] ++ optionals (xorg != null) (with xorg; [ libXrender libXaw libXpm ])
```

Comment:
```suggestion
  ] ++ (with xorg; [ libXrender libXaw libXpm ])
```

Reply:
graphviz still doesn't cross compile so i'll have to make a separate pr for that so i'll do some refactoring in it